pmg/templates/user_management/email_alerts.html

```diff
@@ -25,39 +25,60 @@
       </div>
     </div>
   {% endif %}
+  {% if current_user.is_authenticated() and not current_user.confirmed %}
+    <div class="row">
+      <div class="col-sm-5 col-sm-offset-4">
+        <div class="alert alert-info">
+          <h4>Confirm your email address</h4>
+          <p>Once you confirm your email address, you can subscribe to free email alerts.</p>
+          <br>
+          <p>You were sent an email with confirmation link when you signed up. If you have not received it, you can <a href="{{ url_for('security.send_confirmation', next=request.full_path) }}">request another confirmation email</a>.</p>
```

Comment:
Perhaps "We sent you an email with a confirmation link..." -- active voice is better than passive.